pyproject.toml

```
@@ -1,7 +1,70 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "linode_api4"
+authors = [{ name = "Linode", email = "developers@linode.com" }]
+description = "The official python SDK for Linode API v4"
```

Nitpick from the original metadata but we might as well fix it in this PR 👍 

```suggestion
description = "The official Python SDK for Linode API v4"
```